dvc/commands/checkout.py

```diff
@@ -110,7 +110,7 @@ def add_parser(subparsers, parent_parser):
         nargs="*",
         help=(
             "Limit command scope to these tracked files/directories, "
-            ".dvc files, or stage names."
+            ".dvc files and stage or foreach-group names."
```

Comment:
Copy editing I'd suggest
```suggestion
            ".dvc files, stage or foreach-group names."
```
But TBH I don't think this change is necessary per https://github.com/iterative/dvc.org/pull/3989#pullrequestreview-1118249906. Up to you, cheers.